sql/core/src/main/scala/org/apache/spark/sql/internal/columnNodeSupport.scala

```diff
@@ -237,7 +238,8 @@ private[sql] trait ColumnNodeToExpressionConverter extends (ColumnNode => Expres
   }
 }
 
-private[sql] object ColumnNodeToExpressionConverter extends ColumnNodeToExpressionConverter {
+@DeveloperApi
+object ColumnNodeToExpressionConverter extends ColumnNodeToExpressionConverter {
```

Comment:
Quick question, we have an implicit class in `SparkSession` called `RichColumn`. That will 'restore' the `expr` functionality. Is that something that would work for you? Alternatively, if the use of `SparkSession` is cumbersome, we could also add another implicit class to `org.apache.spark.sql.classic.ClassicConversions`.

Reply:
So for Java users?

Reply:
Sure, I see your point. I would love to meet the Java developer that works with Spark internals :)...

They can still use these objects.